documentation/docs/student/quiz-training-mode.mdx

```diff
@@ -0,0 +1,200 @@
+---
+id: quiz-training
+title: Quiz Training Mode
+sidebar_label: Quiz Training
+---
+
+import trainingTab from "./assets/quiz-training-mode/quiz-training-tab.png";
+import scorecard from "./assets/quiz-training-mode/scorecard.png";
+import mcQuestion from "./assets/quiz-training-mode/mc-question.png";
+import mcQuestionSolution from "./assets/quiz-training-mode/mc-question-solution.png";
+import ddQuestion from "./assets/quiz-training-mode/dd-question.png";
+import saQuestion from "./assets/quiz-training-mode/sa-question.png";
+import saQuestionSolution from "./assets/quiz-training-mode/sa-question-sample-solution.png";
+import unratedNotification from "./assets/quiz-training-mode/unrated-notification.png";
+import description from "./assets/quiz-training-mode/description.png";
+import leaderboard from "./assets/quiz-training-mode/leaderboard.png";
+import userSettings from "./assets/quiz-training-mode/user-settings.png";
+import userLeaderboardSettings from "./assets/quiz-training-mode/user-leaderboard-settings.png";
+import Callout from "../../src/components/Callout/Callout";
+import {CalloutVariant} from "../../src/components/Callout/Callout.types";
+import Image, {ImageSize} from "../../src/components/Image/Image";
+
+# Quiz Training Mode
+
+The Quiz Training Mode allows you to reinforce your knowledge through repeated practice of quiz questions. The system uses a spaced-repetition algorithm to ensure you review questions at optimal intervals.
+
+---
+
+## Getting Started
+
+<Image src={trainingTab} alt={"Description"} size={ImageSize.small}></Image>
+
+When you first visit the Quiz Training, a welcome dialog appears explaining the key features:
+
+### How It Works
+
+Quiz Training is based on the principle of spaced repetition. Questions you answer correctly are repeated at longer intervals, while incorrectly answered questions appear more frequently.
```

Comment:
We are repeating the "spaced repetition" here. Do we really need to mention it twice?

If we do, can we be consistent with the spelling?